### PR TITLE
finished api-registration exercise

### PR DESCRIPTION
**Pull request by SarahChoung**
Screen Shot 2021-01-05 at 9 06 48 PM

### CODE REVIEW

File: api-registration/server/index.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      insert into "users"("username", "hashedPassword")
+      values ($1, $2)
+      returning "username", "hashedPassword", "createdAt"
+    `;
+      const params = [username, hashedPassword];
+      return db.query(sql, params);
+    })
+    .then((result) => {
+      const userInfo = result.rows[0];
+      res.status(201).json(userInfo);
```
Comment (thebearingedge) [inline api-registration/server/index.js:39]: The client should receive the userId, username, and createdAt, but not the hashedPassword.

File: api-registration/server/index.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    })
+    .catch((err) => next(err));
+
+  /**
+   * Hash the user's password with `argon2.hash()`
+   * Then, 😉
+   *   Insert the user's "username" and "hashedPassword" into the "users" table.
+   *   Then, 😉
+   *     Respond to the client with a 201 status code and the new user's "userId", "username", and "createdAt" timestamp.
```
Comment (thebearingedge) [inline api-registration/server/index.js:48]: 😄

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: api-registration/server/index.js (hunk 2 above)
Comment (SarahChoung) [reply api-registration/server/index.js:48]: 🤦‍♀️

Comment (SarahChoung) [reply api-registration/server/index.js:48]: Screen Shot 2021-01-05 at 9 14 55 PM

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**